transforms/code/repo_level_ordering/ray/src/repo_level_order_transform.py

```diff
@@ -293,16 +350,22 @@ def add_input_params(self, parser: ArgumentParser) -> None:
         # in the metadata collected by the Ray orchestrator
         # See below for remove_from_metadata addition so that it is not reported.
         parser.add_argument(
-            f"--{cli_prefix}stage_one_only",
+            f"--{cli_prefix}{stage_one_only_key}",
             action="store_true",
             help="If this flag is set, transform only builds the repo grouping and doesn't write output",
         )
         parser.add_argument(
             f"--{cli_prefix}{grouping_column_key}",
             type=str,
-            default="repo_name",
+            default=repo_column_default_value,
             help="The name of the column which has repo name.",
         )
+        parser.add_argument(
+            f"--{cli_prefix}{language_column_key}",
+            type=str,
+            default="language",
```

Comment:
default should be in a variable, for consistency, like the others.

Reply:
really applies to the bool args as well.  i'm sure this seems like busy work, but its much nicer to be able to browse the top of the file for this type of thing than dig through 100s of lines of code to find the 1 line that sets the default.